profiler/profiler_test.go

```diff
@@ -22,11 +23,28 @@ import (
 	"time"
 
 	"gopkg.in/DataDog/dd-trace-go.v1/internal/log"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/telemetry"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+func setenv(t *testing.T, key, value string) {
+	t.Helper()
+	// re-implemented testing.T.Setenv since that function requires Go 1.17
```

Comment:
Can we not upgrade to 1.17? (Doesn't have to be in this PR of course, but just a meta question)

Reply:
As of yesterday, yes 🎉  Go 1.19 is out now, so according to our [support policy](https://github.com/datadog/dd-trace-go#package-versioning) we can drop Go 1.16 support.